docs/reference/commandline/system_prune.md

```diff
@@ -64,6 +65,16 @@ deleted: sha256:3a88a5c81eb5c283e72db2dbc6d65cbfd8e80b6c89bb6e714cfaaa0eed99c548
 Total reclaimed space: 13.5 MB
 ```
 
+## Filtering
+
+The filtering flag (`-f` or `--filter`) format is of "key=value". If there is more
+than one filter, then pass multiple flags (e.g., `--filter "foo=bar" --filter "bif=baz"`)
+
+The currently supported filters are:
```

Comment:
Please clarify that `filter` is not supported for volumes and networks? (If you are facing a difficulty in supporting them)